waku/v2/node/wakunode2.nim

```diff
@@ -1013,9 +1013,34 @@ when isMainModule:
     
     ok(storeTuple)
 
-  # 2/6 Initialize node
+  # 2/7 Retrieve dynamic bootstrap nodes
+  proc retrieveDynamicBootstrapNodes(conf: WakuNodeConf): Result[seq[RemotePeerInfo], cstring] =
```

Comment:
Is there a reason we are using `cstring` here as opposed to hiding this in a lower level API? Seems like an implementation detail that'd be better to abstract away to either a string or some other relevant type. (I realize this might be a consequence of lower level API)

Reply:
There is no reason. I converted the Result to `SetupResult`, which all other setup methods use as well. This is much clearer.  Thank you :).